src/drivers/adis16448/adis16448.cpp

```diff
@@ -293,6 +293,21 @@ class ADIS16448 : public device::SPI
 		uint16_t		mag_z;
 		uint16_t		baro;
 		uint16_t		temp;
+
+		ADISReport():
+			cmd(0),
```

Comment:
Minor suggestion, initialize these right in the class definition. It's much easier to do a pass and ensure you've got them all, and you don't have to deal with ordering.